java-components/cli/src/main/openapi/tekton-results.yaml

```diff
@@ -0,0 +1,764 @@
+      operationId: delete_result_by_uid
+      summary: Delete a particular result using UID
+      description: ""
+    patch:
+      requestBody:
+        description: The result to be updated.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Result"
+        required: true
+      tags:
+        - Results
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Result"
+          description: Default response
+      operationId: update_result_by_uid
+      summary: Update result given the UID
+      description: ""
+    parameters:
+      - $ref: "#/components/parameters/parent"
+        name: parent
+        x-last-modified: 1677672496911
+      - $ref: "#/components/parameters/result_uid"
+        name: result_uid
+        x-last-modified: 1677672511792
+    x-last-modified: 1677672484697
+  /v1alpha2/parents/{parent}/results/{result_uid}/logs:
+    summary: List logs associated with a result
+    get:
+      tags:
+        - Logs
+      responses:
+        "200":
+          $ref: "#/components/responses/RecordsList"
+          x-last-modified: 1677673949442
+      operationId: list_logs_by_result_uid
+      summary: List Logs given the Result UID
+      description: >-
+        Logs can be read across Results by specifying `-` as the `result_uid` or
+        across parents by specifying `-` as the `parent`. (e.g.,
+        **default/results/-** or **-/results/-**). This can be used to read and
+        filter matching Logs without knowing the exact Result name.
+    parameters:
+      - $ref: "#/components/parameters/parent"
+        name: parent
+        x-last-modified: 1677672653795
+      - $ref: "#/components/parameters/result_uid"
+        name: result_uid
+        x-last-modified: 1677672667367
+      - $ref: "#/components/parameters/filter"
+        name: filter
+        x-last-modified: 1679485318262
+      - $ref: "#/components/parameters/page_size"
+        name: page_size
+        x-last-modified: 1679485331528
+      - $ref: "#/components/parameters/page_token"
+        name: page_token
+        x-last-modified: 1679485343811
+      - $ref: "#/components/parameters/order_by"
+        name: order_by
+        x-last-modified: 1679485356281
+    x-last-modified: 1677672632222
+  /v1alpha2/parents/{parent}/results/{result_uid}/logs/{log_uid}:
+    summary: Get or Delete Logs
+    get:
+      tags:
+        - Logs
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                # STONEBLD-1568 - Log returns a sequence of json objects as a string not a Log. These need
+                #                 to be separately parsed and then reassembled.
+                # See https://github.com/tektoncd/results/blob/main/docs/large-log-files.md
+                # $ref: "#/components/schemas/Log"
+                type: string
```

Comment:
Only change to the schema. From speaking to the tekton-results devs it can return (unexpectedly depending upon the size of the log) either a Log or, effectively, a series of JSON objects formatted as per the large-log-files reference